decimal_test.go

```diff
@@ -27,61 +27,70 @@ func TestNumberAddSub(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		a, err := FromString(test.a)
+		a, err := newDecimal.NewFromString(test.a)
 		assert.NoError(t, err)
-		b, err := FromString(test.b)
+		b, err := newDecimal.NewFromString(test.b)
 		assert.NoError(t, err)
-		sum, err := FromString(test.sum)
+		sum, err := newDecimal.NewFromString(test.sum)
 		assert.NoError(t, err)
 
-		assert.Equal(t, sum, a.Add(b))               // sum = a + b
-		assert.Equal(t, sum, a.Add(b).Add(b).Sub(b)) // sum = a + b + b -b
+		sum1 := a.Add(b)
+		sum2 := a.Add(b).Add(b).Sub(b)
+
+		// Zero should never be compared with == or != directly, please use decimal.Equal or decimal.Cmp instead.
```

Comment:
Should we always use `Equal` then?

Reply:
If talking about `==` and `!=`, `decimal.Equal` seems like more clear solution, but tt will also work with `decimal.Cmp` if needed for some reason.